app/views/servers/_badges.html.erb

```diff
@@ -0,0 +1,53 @@
+<!-- START: badges -->
+<style>
+  /* relevant styles */
+  .img__wrap {
+    position: relative;
+    height: 50px;
+    width:50px;
+    display:inline-block;
+    border-radius:50%;
+  }
+
+  .img__description {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    background: rgba(51,51,51, 0.72);
+    color: #fff;
+    visibility: hidden;
+    opacity: 0;
+    border-radius:50%;
+    height:50px;
+    width:50px;
+    font-size:8px;
+
+    /* transition effect. not necessary */
+    transition: opacity .2s, visibility .2s;
+  }
+
+  .img__wrap:hover .img__description {
+    visibility: visible;
+    opacity: 1;
+  }
+</style>
+<% if true#!@server.badges.empty? %>
+<div class="row" style="height:50px;margin-top:-20px;margin-bottom:20px;text-align:center">
+  <% for badge in Badge.all %>
+  <% if @server.badges.include? badge.id %>
```

Comment:
Could we just do something like ```for badge in @server.badges```.  You may need to just set up the model so that it references the other collection properly... I'm pretty sure you can do that with ruby/mongo.  See some of the other models.